README.md

```diff
@@ -129,6 +129,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Rollback a GitHub Release](https://github.com/author/action-rollback)
 - [Lock Closed Issues and Pull Requests after a Period of Inactivity](https://github.com/dessant/lock-threads)
 - [Get Commit Difference Count Between Two Branches](https://github.com/jessicalostinspace/commit-difference-action)
+- [Generate release notes based on git references](https://github.com/metcalfc/changelog-generator)
```

Comment:
Thank you for the clarification in the commit description. Please use title casing - there is a converter in this list: https://github.com/sdras/awesome-actions/blob/master/contributing.md

Reply:
Fixed.